jupyterlab/handlers/yjs_echo_ws.py

```diff
@@ -52,43 +66,42 @@ async def get(self, *args, **kwargs):
             raise web.HTTPError(403)
         return await super().get(*args, **kwargs)
 
-    def open(self, guid):
-        #print("[YJSEchoWS]: open", guid)
-        cls = self.__class__
+    def open(self, type_path):
+        # print("[YJSEchoWS]: open", type_path)
+        type, path = type_path.split(":", 1)
         self.id = str(uuid.uuid4())
-        self.room_id = guid
-        room = cls.rooms.get(self.room_id)
+        self.room_id = path
+        room = ROOMS.get(self.room_id)
         if room is None:
-            room = YjsRoom()
-            cls.rooms[self.room_id] = room
+            room = YjsRoom(type)
+            ROOMS[self.room_id] = room
         room.clients[self.id] = ( IOLoop.current(), self.hook_send_message, self )
         # Send SyncStep1 message (based on y-protocols)
         self.write_message(bytes([0, 0, 1, 0]), binary=True)
 
     def on_message(self, message):
-        #print("[YJSEchoWS]: message, ", message)
-        cls = self.__class__
+        # print("[YJSEchoWS]: message,", message)
         room_id = self.room_id
-        room = cls.rooms.get(room_id)
+        room = ROOMS.get(room_id)
         if message[0] == ServerMessageType.ACQUIRE_LOCK:
             now = int(time.time())
-            if room.lock is None or now - room.timeout > (10 * len(room.clients)) : # no lock or timeout
+            if room.lock is None or now - room.timeout > (10 * len(room.clients)): # no lock or timeout
```

Comment:
Do we still need room-locks? Or are you keeping them for compatibility sake?

Reply:
Indeed I don't think we need them anymore, I will remove them.